seeds/scripts/rerunners/ddr/README.md

```diff
@@ -28,3 +28,24 @@ File must be stored in the same folder as the script with the name `musicdb.xml`
 
 2) At the last line of the script, eventually change the version if necessary. Current supported versions are `a3` and `konaste` (which are very similar since konaste is based on a3)
 3) Run `ts-node parse-gameData-xml.ts`
+
+# Update stepCounts
+
+1) Download csv files for Single and Double from https://docs.google.com/spreadsheets/d/10NT1VPYV8JHEpi68SBCTApOO5mWEjIAbJP3C5Csthyk/edit?gid=1455718580#gid=1455718580
```

Comment:
Using this spreadsheet is a non-starter, sorry.

I'm against using community sheets for notecount information because they are *almost always subtly wrong*. We've used them in the past and been bitten again-and-again-and-again. This makes this PR unmergable straight away because I do not trust the accuracy of this data.

I took a look around the sheet and a couple of things raise my eyebrows, like the "Chart Analyzer" in the spreadsheet, which looks suspiciously close the analysis done by simply love. My gut instinct is that this is someone converting charts into `.sm` and then using stepmania as the analysis tool. I can tell you firsthand that the analysers in Simply Love are outright incorrect at times, and are not to be trusted. I can also tell you that  `.sm` is a sticky business with a couple mean edge cases that might not be handled well here.

Either way, I can't merge sheet data like this into the site.